### PR TITLE
Copter: GCS message when Surface Tracking enable/disable via rc

### PR DESCRIPTION
**Pull request by Hwurzburg**
provide user feedback on switch changes

### CODE REVIEW

File: ArduCopter/RC_Channel.cpp [hunk 1]
```diff
@@ -252,8 +252,10 @@ void RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const aux_sw
 #if RANGEFINDER_ENABLED == ENABLED
             if ((ch_flag == HIGH) && copter.rangefinder.has_orientation(ROTATION_PITCH_270)) {
                 copter.rangefinder_state.enabled = true;
+                copter.gcs().send_text(MAV_SEVERITY_WARNING, "SurfaceTracking enabled");
```
Comment (peterbarker) [inline ArduCopter/RC_Channel.cpp:255]:
```suggestion
                gcs().send_text(MAV_SEVERITY_WARNING, "SurfaceTracking enabled");
```

gcs() should be available from any context

**Comment by rmackay9** [comment]
By the way, i'm happy with this change.  In general I hope we can keep the number of send-text messages sent to the GCS to a minimum especially when things are going fine... still, I think sending messages so that the user is aware that an action they've requested as actually been accepted is ok.

File: ArduCopter/RC_Channel.cpp (hunk 1 above)
Comment (Hwurzburg) [reply ArduCopter/RC_Channel.cpp:255]: was just copying it from the surface tracking module I was also using at one point for messaging while doing rf work....did not know why it had to specify copter either, but it worked....will change...

File: ArduCopter/RC_Channel.cpp [hunk 2]
```diff
@@ -252,8 +252,10 @@ void RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const aux_sw
 #if RANGEFINDER_ENABLED == ENABLED
             if ((ch_flag == HIGH) && copter.rangefinder.has_orientation(ROTATION_PITCH_270)) {
                 copter.rangefinder_state.enabled = true;
+                gcs().send_text(MAV_SEVERITY_WARNING, "SurfaceTracking enabled");
```
Comment (khancyr) [inline ArduCopter/RC_Channel.cpp:255]: using rangefinder maybe ? As this insn't only for surface tracking (landing, precland, etc.)

Comment (Hwurzburg) [reply ArduCopter/RC_Channel.cpp:255]: originally thought of that , but this is only for the downward facing,ground/sruface tracking rangefinder....and you actually are tracking the surface during landing,etc.....but open to anything better that doesn't imply other rangefinders...

**Comment by Hwurzburg** [comment]
replaced by #14509 and #14523

**Pull request closed without merging**